phpstan.neon

```diff
@@ -0,0 +1,22 @@
+includes:
```

Comment:
is this file already blocked by the .htaccess ?

Reply:
Yes @carakas